### ISSUE TITLE
Need Dynamic Aggregate Rating

### ISSUE THREAD

**Issue by mohammed-qutubuddin** [ahmedkaludi/schema-and-structured-data-for-wp#1393]
Reference Ticket URL: https://secure.helpscout.net/conversation/1495869521/192969?folderId=4409869

Get dynamic aggregate rating from google reviews using place id.

**Comment by drewclifton** [comment]
Yes need this please and thanks! This is a "Schema and Structured Data for WP" - but displays front end reviews dynamically... and yet we can't get it in the JSON LD structured data? aggregateRating changes weekly for our clients so we can't keep going back and updating it manually all the time...

I will add that really this seems to be two different plugins smushed together. This plugin should just focus on "Schema and Structured Data for WP" (as per plugin name), and leave front end reviews to another plugin that you can offer separately. We don't need the extra plugin and database size, administrative complexity and additional features - as well as having to wait on getting improvements to the true "Schema and Structured Data for WP" features because front end features are being worked on.

Thanks for receiving my feedback regarding this! And I hope we can get **aggregateRating** via the Places API into our structured data soon. 😀

**Comment by drewclifton** [comment]
Adding for clarity:

We just need data for the fields that are already available in settings, ie:

- Rating (eg 4.8)
- Number of Reviews (eg 64)

Eg as per this option:

![Screen Shot 2021-05-17 at 10 19 51 AM](https://user-images.githubusercontent.com/33555978/118530413-7c182080-b6f9-11eb-97f6-51a265829a78.png)

But could be a global setting and then just check the box when creating schema for homepage so it will pull in I guess ...?

Thanks again.
